src/Apps/Auction/__tests__/AuctionApp.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { routes_AuctionQueryResponse } from "__generated__/routes_AuctionQuery.graphql"
+import { routes } from "Apps/Auction/routes"
+import { createMockNetworkLayer2 } from "DevTools"
+import { createRender } from "found"
+import { Resolver } from "found-relay"
+import getFarceResult from "found/lib/server/getFarceResult"
+import { Environment, RecordSource, Store } from "relay-runtime"
+import { SaleAuction } from "../../__tests__/Fixtures/Sale"
+
+interface MockResponse {
+  sale: routes_AuctionQueryResponse["sale"]
+}
+
+describe("AuctionApp", () => {
+  describe("redirects", () => {
+    async function render(url, mockData) {
+      const network = createMockNetworkLayer2({ mockData })
+      const source = new RecordSource()
+      const store = new Store(source)
+      const environment = new Environment({ network, store })
+
+      return await getFarceResult({
+        url,
+        routeConfig: routes,
+        resolver: new Resolver(environment),
+        render: createRender({}),
+      })
+    }
+
+    const mockResolver = (data: object): MockResponse => ({
+      sale: data as routes_AuctionQueryResponse["sale"],
+    })
```

Comment:
@dleve123 @damassi Does this help? :man_shrugging: 

I'm not too familiar with typing mocked responses...

Reply:
You'll want to type the fixture as [seen here](https://github.com/artsy/reaction/blob/7d10eca854227f1cc47cc190c430015cc4281cc6/src/Apps/Artwork/Components/Seo/__tests__/SeoDataForArtwork.fixture.ts#L3)